server/src/main/java/org/elasticsearch/cluster/coordination/ClusterBootstrapService.java

```diff
@@ -126,6 +128,8 @@ void logBootstrapState(Metadata metadata) {
             final var clusterUUID = metadata.clusterUUID();
             if (bootstrapRequirements.isEmpty()) {
                 logger.info("this node is locked into cluster UUID [{}] and will not attempt further cluster bootstrapping", clusterUUID);
+            } else if (singleNodeDiscovery) {
+                logger.info("this node is locked into single-node cluster UUID [{}]", clusterUUID);
```

Comment:
I think we could reasonably have the same message in both cases:

```suggestion
            if (singleNodeDiscovery || bootstrapRequirements.isEmpty()) {
                logger.info("this node is locked into cluster UUID [{}] and will not attempt further cluster bootstrapping", clusterUUID);
```

At least we should include the `and will not attempt further cluster bootstrapping` bit in both messages.